common.sh

```diff
@@ -13,6 +13,7 @@ export LAMBCI_CHECKOUT_BRANCH="${LAMBCI_CHECKOUT_BRANCH:-$LAMBCI_BRANCH}"
 export LAMBCI_DOCKER_CMD="${LAMBCI_DOCKER_CMD:-$SCRIPT_DIR/runbuild.sh}"
 export LAMBCI_DOCKER_FILE="${LAMBCI_DOCKER_FILE:-Dockerfile.test}"
 export LAMBCI_DOCKER_TAG="${LAMBCI_DOCKER_TAG:-lambci-ecs-${LAMBCI_REPO}:${LAMBCI_COMMIT}}"
+export LAMBCI_DOCKER_TAG=${LAMBCI_DOCKER_TAG,,} # to lower case
```

Comment:
Oh nice – thanks! Can you surround with quotes too? (I mean, it shouldn't have spaces... but better to be consistent, if only so that errors should be a little nicer)